community/flatpak-builder/APKBUILD

```diff
@@ -26,22 +23,18 @@ build() {
 		--sbindir=/usr/bin \
 		--libexecdir=/usr/lib/$pkgname \
 		--disable-static \
-		--disable-documentation \
 		--with-dwarf-header=/usr/include/libdwarf
 
 	make
 }
 
 check() {
-	cd "$builddir"
 	#make -k check
 	./flatpak-builder --version
```

Comment:
@clandmeter asked me to not do such simplistic checks to not give a false sense of proper testing being done.

Reply:
Sure, but wasn't sure if I should change existing behavior here.